package.json

```diff
@@ -17,6 +17,9 @@
     "test:unit": "npx nightwatch tests/unit_tests",
     "eslint": "eslint src"
   },
+  "overrides": {
+    "semver": "^7.5.3"
+  },
```

Comment:
Instead of using this, just install the latest version of `nightwatch`. That should fix it.

Reply:
I didn't get like install latest nightwatch version?
It will be according to version mentioned in package.json
can you help out a little for this

Reply:
Yeah, just do `npm i -D nightwatch@latest` and `npm un -D @types/nightwatch` (types are automatically included in the latest version of Nightwatch).

Reply:
Thank you , I have added the changes with resolving for one more moderate package issue

Reply:
After this severe issues are remaining ,on updating which breaking changes are occuring